src/transaction-hooks.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('feathers-knex-transaction');
+
+export const start = (options) => {
+  const { dbServiceName } = options;
+  debug('started transaction system with %s service', dbServiceName);
+  return (hook) =>
+    new Promise(resolve =>
+      hook.app.get(dbServiceName).transaction(trx => {
```

Comment:
Can you try if `hook.service.Model` works here? I think it should (because it is the same thing really).